Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityToolkitFiles.cs

```diff
@@ -59,6 +59,23 @@ public static void OnPostprocessAllAssets(string[] importedAssets, string[] dele
             new Dictionary<MixedRealityToolkitModuleType, HashSet<string>>();
         private readonly static Task searchForFoldersTask;
 
+        /// <summary>
+        /// Resolves the given asset to its full path if and only if the asset belongs to the
+        /// Assets folder (i.e. it is prefixed with "Assets/..."
+        /// </summary>
+        /// <remarks>
+        /// If not associated with the Assets folder, will return the path unchanged.
+        /// </remarks>
+        private static string ResolveFullAssetsPath(string path)
+        {
+            if (path.StartsWith("Assets"))
```

Comment:
Add null check for safe measure?

Reply:
I think I'll keep this one as-is - if we did do some null checking, I wouldn't want to proliferate this into all of the private static helpers, and would rather do the checking at the various data junction points (i.e. the actual Unity API). Even in this case, however, I assume that we should be able to depend on strings from Unity APIs being not null, in which case we would only null check if there were a Unity bug)